docs/operating/manuals/collaset.md

```diff
@@ -436,4 +434,125 @@ NAME                     READY   STATUS    RESTARTS     AGE
 collaset-sample-5wvlh    1/1     Running   1 (6s ago)   2m10s
 collaset-sample-ldvrg    1/1     Running   1 (6s ago)   2m10s
 collaset-sample-pbz75    1/1     Running   1 (6s ago)   2m10s
-```
\ No newline at end of file
+```
+
+### Replace Update Pod
+
+CollaSet provides the `Replace` policy for certain applications that are sensitive to the available number of Pods.
+
+``` yaml
+apiVersion: apps.kusionstack.io/v1alpha1
+kind: CollaSet
+metadata:
+  name: collaset-sample
+spec:
+  ...
+  updateStrategy:
+    podUpgradePolicy: Replace  # Options: InPlaceIfPossible, ReCreate, Replace
+```
+
+The `Replace` policy indicates that CollaSet should update a Pod by creating a new one to replace it. 
+Unlike the `Recreate` policy, which deletes the old Pod before creating a new updated one, or the `InPlaceIfPossible` policy, which updates the current Pod in place, 
```

Comment:
`Recreate` or `ReCreate`?  In operating defined `CollaSetRecreatePodUpdateStrategyType PodUpdateStrategyType = "ReCreate"`. But I believe the correct way to write it is `Recreate`.
I